### PR TITLE
theme-designer: theme designer tokens

### PR DESCRIPTION
**Pull request by emmayjiang**
very basic search working for theme designer tokens.

leaving as a draft for now until micah can look at my PR for the theme designer tokens themselves :)

image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/b01ac1a166a8dc67a6c53409d9fe86977517bae6'>b01ac1a166a8dc67a6c53409d9fe86977517bae6</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-components/theme-designer/src/components/TokenBoxes/TokenBoxes.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { TokenBoxes } from './TokenBoxes';
+export default { component: TokenBoxes };
+export const Default = {};
```
Comment (micahgodbolt) [inline packages/react-components/theme-designer/src/components/TokenBoxes/TokenBoxes.stories.tsx:3]: we CAN come back to the stories later, but note that this TokenBoxes story doesn't render without brandColors passed into it.

**Comment by emmayjiang** [comment]
This PR is now for theme designer token changes, search will be in a new PR

**Comment by emmayjiang** [comment]
image

**Pull request merged**